core/server/log.js

```diff
@@ -30,22 +29,11 @@ module.exports = function log(...msg) {
   console.log(d, ...msg)
 }
 
-const throttledConsoleError = throttle(console.error, 10000, {
-  trailing: false,
-})
-
-module.exports.error = function error(...msg) {
+module.exports.error = function error(err) {
   const d = date()
-  listeners.forEach(f => f(d, ...msg))
-  Raven.captureException(msg, sendErr => {
-    if (sendErr) {
-      throttledConsoleError(
-        'Failed to send captured exception to Sentry',
-        sendErr.message
-      )
-    }
-  })
-  console.error(d, ...msg)
+  listeners.forEach(f => f(d, err))
+  Sentry.captureException(err)
```

Comment:
Calling this with the error instead of an array containing the error fixes the problem where the stack trace was truncated.